conftest.py

```diff
@@ -146,19 +147,32 @@ def pytest_sessionfinish(session: pytest.Session, exitstatus):  # pylint: disabl
 def default_configuration():
     """Return the default test compilation configuration."""
 
-    # Remove parameter_selection_strategy once it is set to multi-parameter in Concrete Python
```

Comment:
please don't remove these comments as they are pretty important to track down known issues

Reply:
Correct ! I removed them by mistake